Sources/GRPCHTTP2TransportNIOPosix/HTTP2ClientTransport+Posix.swift

```diff
@@ -69,8 +69,8 @@ extension HTTP2ClientTransport {
     /// - Throws: When no suitable resolver could be found for the `target`.
     public init(
       target: any ResolvableTarget,
+      config: Config,
```

Comment:
Nit: can we reorder the param in the docs too?

Reply:
Absolutely, good catch!